### PR TITLE
news post for MEI Basic

### PR DESCRIPTION
**Pull request by kepper**
news post introducing MEI Basic

### CODE REVIEW

File: _posts/2019-05-31-mei-basic-first-draft.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title:  "First Draft of MEI Basic"
+date:   2019-05-31 07:30:00
+categories: update
+---
+
+The MEI Community is pleased to announce the first draft version of MEI Basic, 
+a restricted profile of MEI intended to support data interchange both
```
Comment (ahankinson) [inline _posts/2019-05-31-mei-basic-first-draft.md]: "... a simplified profile..." ?

File: _posts/2019-05-31-mei-basic-first-draft.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+within MEI and with other encoding formats.  
+
+MEI Basic has been discussed at several occasions over the past three years.
+It intends to reduce MEI so that every feature can only expressed in one
```
Comment (ahankinson) [inline _posts/2019-05-31-mei-basic-first-draft.md]: "... intends to simplify MEI..."

File: _posts/2019-05-31-mei-basic-first-draft.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+specific way, and also removes all editorial or analytical possibilities, as
+well as the integration of facsimiles and recordings. What is left is a fairly
```
Comment (ahankinson) [inline _posts/2019-05-31-mei-basic-first-draft.md]: Remove 'fairly'

**Comment by ahankinson** [comment]
Will wait for @musicEnfanthen to have a look

File: _posts/2019-05-31-mei-basic-first-draft.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+date:   2019-05-31 07:30:00
+categories: update
+---
+
+The MEI Community is pleased to announce the first draft version of MEI Basic, 
+a simplified profile of MEI intended to support data interchange both
+within MEI and with other encoding formats.  
+
+MEI Basic has been discussed at several occasions over the past three years.
+It intends to simplify MEI so that every feature can only expressed in one
```
Comment (musicEnfanthen) [inline _posts/2019-05-31-mei-basic-first-draft.md]:
```suggestion
It intends to simplify MEI so that every feature can only be expressed in one
```

**Pull request merged**